runtime/lib/ttnn/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set_property(TARGET TTRuntimeTTNN PROPERTY CXX_STANDARD 20)
 target_include_directories(TTRuntimeTTNN PUBLIC
   ${PROJECT_SOURCE_DIR}/runtime/include
   ${PROJECT_BINARY_DIR}/include/ttmlir/Target/Common
+  ${TTMLIR_SOURCE_DIR}/third_party/tt-metal/src/tt-metal-build/include
```

Comment:
Should this be added to `TTMETAL_INCLUDE_DIRS`

Reply:
I'm not sure, I added it here as none of the `INCLUDE_DIRS` pointed to the `tt-metal-build` dir, so I was unsure if that was intentional or not.

Reply:
So if you look at the next line `target_include_directories(TTRuntimeTTNN SYSTEM PUBLIC "$<BUILD_INTERFACE:${TTMETAL_INCLUDE_DIRS}>")`

I was thinking `TTMETAL_INCLUDE_DIRS` might be a better place to add this as it points to tt-metal

Reply:
Ah yeah I get what you're saying, I meant to say that none of the directories within `TTMETAL_INCLUDE_DIRS` actually points to `tt-metal-build` so I was unsure if I should edit the variable with this new include I'm adding.

Reply:
Yeah adding it to the variable should be fine, that way anywhere else we want to log with tracy they'll just be able to do it.